### PR TITLE
Neos 9 Support

### PR DESCRIPTION
**Pull request by samsauter**
Adding Neos 9 compatibilty.

- Fixing deprecated fusion prototypes

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# General swap/backup files
+*.so
+*.log
+*.out
+*~
+*.swp
+*.DS_Store
+*.sass-cache*
+
+!*.gitkeep
```
Comment (bwaidelich) [inline .gitignore:22]: Please ignore these in your local working copy (`vi .git/info/exclude`)

File: Classes/Eel/Helper/FormHelper.php [hunk 2]
```diff
@@ -207,4 +207,4 @@ public function allowsCallOfMethod($methodName): bool
     {
         return true;
     }
-}
\ No newline at end of file
+}
```
Comment (bwaidelich) [inline Classes/Eel/Helper/FormHelper.php:210]: ?

**Review by bwaidelich — COMMENTED** [review]
nice, thanks. Just a comment re .gitignore and needless whitespace change

**Comment by dlubitz** [comment]
Hey @samsauter could you please check the comments from @bwaidelich? Let me know, if you are busy, than I could take this over, to get it merged.

**Comment by dlubitz** [comment]
Actually its a duplicate.
I would close this in favor https://github.com/neos/form-fusionrenderer/pull/29 and move your additional changes there.

Thank you @samsauter

**Pull request closed without merging**